<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2019 - 2021 by Pedro Mendes, Rector and Visitors of the </comment>
<comment> University of Virginia, University of Heidelberg, and University </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2017 - 2018 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and University of </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2010 - 2016 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 - 2009 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2004 - 2007 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc. and EML Research, gGmbH. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CMCAResultSubwidget</class>
 <widget class="CopasiWidget" name="CMCAResultSubwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCA Result Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mpLblResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Metabolic Control Analysis Result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSaveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save to File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="mTopLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComboScale">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>scaled</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unscaled</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="mTabWidget">
     <widget class="QWidget" name="tab0">
      <attribute name="title">
       <string>Elasticities</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="CQArrayAnnotationsWidget" name="mpArrayElasticities" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Flux Control Coefficients</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="CQArrayAnnotationsWidget" name="mpArrayFCC" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>Concentration Control Coefficients</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="CQArrayAnnotationsWidget" name="mpArrayCCC" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQArrayAnnotationsWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/CQArrayAnnotationsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">copasi/UI/CQArrayAnnotationsWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mComboScale</sender>
   <signal>activated(int)</signal>
   <receiver>CMCAResultSubwidget</receiver>
   <slot>slotScaled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>CMCAResultSubwidget</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
  <slots>
    <slot>slotSave()</slot>
    <slot>slotScaled()</slot>
  </slots>
</ui>
